0 program TryCatchDemo {
0     func divide(x, y) {
2         let result = 0;
5         try {
7             result = x / y;
11         } catch {
13             result = -1;
16         }
16         return result;
19     }
19 
19     main {
20         let a = 10;
23         let b = 0;
25         let c = 0;
27         c = divide(a, b);
33         output(c);
35 
35         b = 2;
37         c = divide(a, b);
43         output(c);
45 
45         try {
47             c = a / b;
51             output(c);
53         } catch {
55             output(-2);
58         }
58 
58 
58         output(0);
61     }
61 }
62 
===Parsing success!===
